File: web2-Project/src/app/entities/message/message.ts

export class Message {
    id: number;
    myId: string; 
    hisId: string;
    text: string; // sadrzaj poruke
    dateTime: Date; // vreme kad je poslata
    isUnread: boolean; // da li je procitana

    constructor(myId: string, text: string, date: Date, isUnread: boolean) {
        this.myId = myId;
        this.text = text;
        this.dateTime = date;
        this.isUnread = isUnread;
    }
}

File: MAANPP20/MAANPP20/Common/Models/Common/Message.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Common.Models.Common
{
	public class Message
	{
		[Key]
		public int id { get; set; }

		public string text { get; set; }

		public DateTime dateTime { get; set; }

		public string myId { get; set; }

		public string hisId { get; set; }

		public bool isUnread { get; set; }

		public bool deleted { get; set; }

		public Message()
		{
			deleted = false;
		}
	}
}
